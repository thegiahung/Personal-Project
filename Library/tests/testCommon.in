Common 100000 100001
Add member Python
Add member Java
Common 100000 100001
Add Collection books.csv
Common 100000 100001
Common 100000 100000 100001
rent 100000 111111
relinquish 100000 111111
rent 100001 111111
relinquish 100001 111111
Common 100000 100001
exit
