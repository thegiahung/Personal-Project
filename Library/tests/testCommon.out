user: No members in system.

user: Success.

user: Success.

user: No books in system.

user: 17 books successfully added.

user: No common books.

user: Duplicate members provided.

user: Success.

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)

user: Ending Library process.
